app/models/user.rb

```diff
@@ -65,24 +65,20 @@ def self.tenant_id_clause(user_or_group)
     {table_name => {:id => users_ids}}
   end
 
-  def self.in_region
+  def self.in_my_region_column
     where(:region => my_region_number)
   end
 
-  def self.in_my_region
```

Comment:
I think `in_region` is there to query a user from another region. Not sure the use case and if this is even needed.